### PR TITLE
Wazuh changes

### PR DESCRIPTION
**Pull request by arturo-seijas**
* Change to wazuh snap
* Minimal documentation changes

### CODE REVIEW

File: tests/integration/upgrades/test_small_deployment_upgrades.py [hunk 1]
```diff
@@ -102,13 +102,15 @@ async def _build_env(ops_test: OpsTest, version: str) -> None:
 @pytest.mark.group("happy_path_upgrade")
 @pytest.mark.abort_on_fail
 @pytest.mark.skip_if_deployed
+@pytest.mark.skip
```
Comment (javierdelapuente) [inline tests/integration/upgrades/test_small_deployment_upgrades.py:105]: Maybe a reason stating the reason for commenting this test (same for the others in this file?

Comment (arturo-seijas) [reply tests/integration/upgrades/test_small_deployment_upgrades.py:105]: I intend to unskip in a follow up PR. This is an upgrade test

**Review by cbartz — COMMENTED** [review]
The changes look good to me as far as I can judge (without any knowledge on the project). Have a small comment. Can you add a label to the PR. Will the pull request template be updated to match our team's standard?

File: tests/integration/upgrades/test_small_deployment_upgrades.py (hunk 1 above)
Comment (arturo-seijas) [reply tests/integration/upgrades/test_small_deployment_upgrades.py]: Done

File: tests/integration/plugins/test_plugins.py [hunk 2]
```diff
@@ -368,6 +368,7 @@ async def test_knn_enabled_disabled(ops_test, deploy_type: str):
 
 @pytest.mark.parametrize("deploy_type", SMALL_DEPLOYMENTS)
 @pytest.mark.abort_on_fail
+@pytest.mark.skip
```
[the remark this answers is not in the document]
Comment (arturo-seijas) [reply tests/integration/plugins/test_plugins.py]: Done

**Comment by arturo-seijas** [comment]
> The changes look good to me as far as I can judge (without any knowledge on the project). Have a small comment. Can you add a label to the PR. Will the pull request template be updated to match our team's standard?

Added. Not for now. I'm trying to keep the delta from upstream to a minimum. Btw, going forward, we might provide a mechanism to change the workload of a machine charm, so we might change how we build this charm

**Review by javierdelapuente — APPROVED** [review]

**Review by cbartz — APPROVED** [review]

**Pull request merged**